src/client/driver/command-executors/execute-action-command.js

```diff
@@ -168,7 +180,14 @@ export default function executeActionCommand (command, elementAvailabilityTimeou
 
                 ensureCommandArguments(command);
 
-                return createAutomation(elements, command).run();
+                var automationRun = createAutomation(elements, command).run();
+
+                if (command.type === COMMAND_TYPE.uploadFile) {
+                    return automationRun
+                        .catch(filePaths => Promise.reject(new ActionCanNotFindFileToUpload(filePaths)));
```

Comment:
I'd rather move it to `createAutomation` where we already have conditions for command types